plugins/osdn/ovs/proxy.go

```diff
@@ -65,8 +65,12 @@ func (proxy *ovsProxyPlugin) watchPods() {
 
 		switch eventType {
 		case watch.Added, watch.Modified:
+			glog.V(5).Infof("Watch Added or Modified event for Pod %q", pod.ObjectMeta.Name)
```

Comment:
any reason to not do `"Watch %s event...", string(eventType)`? (Or even `strings.Title(string(eventType))`)